### PR TITLE
Draft tidyverse 1.3.0 post

### PR DESCRIPTION
**Pull request by batpigandme**

### CODE REVIEW

**Comment by jennybc** [comment]
Apologies if I'm reviewing something that is already slated for some expansion.

But I think it would be good to have one paragraph reiterating the intended usage of the tidyverse metapackage vs. individual packages, maybe with a link to this post:

https://www.tidyverse.org/blog/2018/06/tidyverse-not-for-packages/

**Comment by batpigandme** [comment]
@jennybc I added a line about this to the intro, but, alternatively, we could expand the "welcome to the tidyverse" section and discuss there some of the design principles (including its intended use).

File: content/blog/2019-11-tidyverse-1-3-0/index.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+```{r message=FALSE, warning=FALSE}
+library(tidyverse)
+```
+
+## Welcome to the Tidyverse
+
+The most significant update in this version is a new vignette, ["Welcome to the Tidyverse"](https://tidyverse.tidyverse.org/articles/paper.html), which is a mirror of the recently-released paper of the same name in the [Journal of Open Source Software](https://joss.theoj.org/papers/10.21105/joss.01686). 
+The (long) list of authors includes all members of the tidyverse organisation and its component packages, and will become the canonical way to cite tidyverse packages.
+
+## Citing the tidyverse
```
[the remark this answers is not in the document]
Comment (batpigandme) [reply content/blog/2019-11-tidyverse-1-3-0/index.Rmarkdown]: 10-4, I'll make them the same level of header then, too

File: content/blog/2019-11-tidyverse-1-3-0/index.Rmarkdown [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+And, as always, attach the package by running:
+
+```{r message=FALSE, warning=FALSE}
+library(tidyverse)
+```
+
+## Welcome to the Tidyverse
+
+The most significant update in this version is a new vignette, ["Welcome to the Tidyverse"](https://tidyverse.tidyverse.org/articles/paper.html), which is a mirror of the recently-released paper of the same name in the [Journal of Open Source Software](https://joss.theoj.org/papers/10.21105/joss.01686).
```
[the remark this answers is not in the document]
Comment (batpigandme) [reply content/blog/2019-11-tidyverse-1-3-0/index.Rmarkdown]: Agreed. Now this is the only thing under _just_ Citing the tidyverse, and the when and how are subsections thereof.

**Pull request merged**